benchmarks/README.md

```diff
@@ -0,0 +1,68 @@
+# Benchmarking LLM Performance: Multi-Round QA Use Case
+
+## Overview
+
+This repository contains benchmarking tools for evaluating the performance of language models in various scenarios. The initial focus of this benchmark is on the multi-round QA (Question Answering) use case. The script `multi-round-qa.py` simulates multiple users interacting with a language model concurrently, allowing you to analyze the model's response quality, throughput, and latency.
```

Comment:
We should add a comparison with baseline vLLM in every performance comparison as well.

Reply:
Do you mean we should post some figures at somewhere periodically?